AutoRecordGenerationExternalModule.php

```diff
@@ -285,7 +285,7 @@ private function handleDestinationProject($record, $event_id, $destinationProjec
 								/* $group_id = */ NULL,
 								/* $survey_hash = */ NULL,
 								/* $response_id = */ NULL,
-								/* $repeat_instance = */ NULL
+								/* $repeat_instance = */ 1
```

Comment:
I know I approved this but it was bouncing around in my skull; I can't remember _why_ I explicitly added this line anymore unfortunately let alone why I used `NULL`, but I wonder if it might be more appropriate to use the `$repeat_instance` param that's provided to this function, especially since that also defaults to 1.

Reply:
@ChemiKyle - 1000% yes.  I didn't see that `handleDestinationProject` had a default project input. We should 100% be using that here. Updated!